EXAMPLES.md

```diff
@@ -64,52 +122,108 @@ and ```secretKey.value``` (SecretKey)
 
 ### Generate account access key
 
-```sh
+The following example shows how to generate a new account access key for the account
+named ```john```
 
+```sh
 vaultclient generate-account-access-key --name john
+```
 
+Output example:
+
+```json
+{
+    "id": "ERUY93RPJHWGUBF0IDQ8",
+    "value": "5Of4usgFeLVH5Qr0NU6UxW0nyeYzdNQaJKQS=dRt",
+    "createDate": "2019-12-26T10:50:02Z",
+    "lastUsedDate": "2019-12-26T10:50:02Z",
+    "status": "Active",
+    "userId": "235437388852"
+}
 ```
 
-The previous example will generate a new account access key for the account
-named ```john```
+### Inject accesskey and secretkey
 
-Output example :
+The following example shows how to inject the supplied account access key
+and secret key for the account named ```john```
 
-```json
+```sh
+vaultclient generate-account-access-key --name john \
+--accesskey=AKIA5X47K766FWESB7PL \
```

Comment:
Seeing a few places with credentials - are these all dummy credentials?

Reply:
notice below there is a credential that is '*****EXAMPLE'

Reply:
Yes, these are dummy credentials. I agree with your comment below for `README.md` changing administrative credentials to <administrative access...>
But in `EXAMPLES.md`, do you think it's okay to provide a dummy key as we are giving output examples? We are also providing a username and timestamp and other details.

Reply:
Hm, I guess if it's a dummy credential it wouldn't matter here then. Thanks for the fixes, reapproving now